cypress/e2e/supervision/deputies/add-deputy.cy.js

```diff
@@ -0,0 +1,93 @@
+beforeEach(() => {
+  cy.loginAs("Case Manager");
+  cy.createAClient();
+  cy.get("@clientId").then((clientId) => cy.createOrderForClient(clientId));
+  cy.get("@clientId").then((clientId) => {cy.visit(`/supervision/#/clients/${clientId}`)});
+});
+
+describe(
+  "Create deputy for client",
+  { tags: ["@supervision", "@deputy", "supervision-core"] },
+  () => {
+    it("Given I'm a Case Manager on Supervision, I can add a deputy to a clients order using only mandatory fields. This deputy will then be selected as main correspondant and fee payer.", () => {
+      cy.get("@clientId").then((clientId) => {
+        cy.searchForADeputyToReachAddADeputyPage();
+        cy.contains("Lay").click();
+        cy.get('[label="First name"]').type("Patrick");
+        cy.get('[label="Last name"]').type("Star");
+        cy.contains("Save & continue").click();
+        cy.waitForStableDOM();
+        cy.get('.footer > .dotted-link').click();
+        cy.get('.TABS_DEPUTIES').click();
+        cy.get('tr.summary-row > :nth-child(1) > .dotted-link').click();
+        cy.get('.person-name').should('contain.text', "Patrick Star");
+        cy.get('.summary-row.open > :nth-child(1)').should('contain.text', "Patrick Star");
+        cy.get('.deputy-details-type').should('contain.text', "Lay");
+        cy.get('.deputy-relation-type').should('contain.text', "Lay");
+        cy.get('.fee-payer').should('be.visible');
+        cy.get('.main-contact').should('be.visible');
+        cy.get('.order-details-main-correspondent').should('contain.text', 'Yes');
+        cy.get('.order-details-fee-payer').should('contain.text', 'Yes');
+      });
+    });
+
+    it("Greys out save and continue button when mandatory form fields not filled", () => {
+      cy.get("@clientId").then((clientId) => {
+        cy.searchForADeputyToReachAddADeputyPage();
+        cy.contains("Lay").click();
+        cy.get(':nth-child(1) > .radio-button').click();
+        cy.get('[label="First name"]').type("Squidward");
+        cy.get('.footer > :nth-child(1) > .button').should('be.disabled');
+      });
+    });
+
+    it("Does not allow over 200 characters in occupation field and shows validation summary", () => {
```

Comment:
Did you check if we have these limits tested somewhere in the jest/component tests? Not sure it needs to be part of the cypress test.

Reply:
No I didn't I just copied it over from the existing protractor ones, happy to remove it if it seems redundant.